integrated_channels/blackboard/client.py

```diff
@@ -40,15 +45,235 @@ def __init__(self, enterprise_configuration):
         self.config = apps.get_app_config('blackboard')
         self.session = None
         self.expires_at = None
+        self.create_course_url = "{}/learn/api/public/v3/courses".format(
+            self.enterprise_configuration.blackboard_base_url
+        )
 
     def create_content_metadata(self, serialized_data):
-        """TODO"""
+        """
+        Create a course from serialized course metadata
+        Returns: (int, str) Status code, Status message
+        """
+        channel_metadata_item = json.loads(serialized_data.decode('utf-8'))
+        if 'externalId' not in channel_metadata_item:
```

Comment:
Don't you have a static method for this?

Reply:
yes thank you!